Source/SideMenuController+SideOver.swift

```diff
@@ -141,16 +141,16 @@ extension SideMenuController {
             self.centerPanelOverlay.alpha = alpha
             self.set(statusUnderlayAlpha: alpha)
             self.sidePanel.frame = destinationFrame
-            }, completion: { _ in
+            }, completion: { () in
```

Comment:
I think you can leave `() in` out completely in Swift 4 😊

Reply:
Why does it even migrate to this then? 🤔 But yes, you're right.

Reply:
I asked myself the same when you noticed all the unnecessary enum prefixes in my PRs 😄